x-pack/plugin/security/qa/service-account/src/javaRestTest/java/org/elasticsearch/xpack/security/authc/service/ServiceAccountIT.java

```diff
@@ -154,11 +154,13 @@ public class ServiceAccountIT extends ESRestTestCase {
                 {
                     "names": [
                         "search-*",
+                        ".elastic-analytics-collections",
                         ".ent-search-*",
                         ".monitoring-ent-search-*",
                         "metricbeat-ent-search-*",
                         "enterprise-search-*",
                         "logs-app_search.analytics-default",
+                        "logs-elastic_analytics.events-*",
```

Comment:
Given the index name is `logs-elastic_analytics.events-{collection_name}-{date}`, would it better to have it like `logs-elastic_analytics.events-*-*`? Or are there other use cases which require a slightly broader index pattern?

Reply:
Makes sense. Thank you!